src/TwigComponent/src/Test/RenderedComponent.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Test;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class RenderedComponent implements \Stringable
```

Comment:
I added this stringable object to wrap the rendered html so we can possibly add something like described in https://github.com/symfony/ux/issues/818#issuecomment-1527750014 in the future.

I wasn't sure if changing the type-hint from string -> stringable object as the return type of `InteractsWithTwigComponent::renderTwigComponent()` would be considered a BC break. If not, I can change to just return `string`.

Reply:
Let's start with this - easier to remove this later if we end up not needing it or it's a pain for some reason than to go in the other direction.